spt/features/aim.cpp

```diff
@@ -177,6 +205,58 @@ CON_COMMAND(tas_aim, "Aims at a point.")
 	}
 }
 
+CON_COMMAND(tas_aim_pos, "Aims at a position. Usage: tas_aim_pos <x> <y> <z> [ticks]")
+{
+	if (args.ArgC() < 4)
+	{
+		Msg("Usage: tas_aim_pos <x> <y> <z>\n");
+		return;
+	}
+
+	int frames = -1;
+	if (args.ArgC() == 5)
+		frames = std::atoi(args.Arg(4));
```

Comment:
frames argument here isn't included in the help text

Reply:
frames argument means the ticks.